src/content/chainlink-local/build/ccip/foundry/local-simulator.mdx

```diff
@@ -0,0 +1,253 @@
+
+   ```solidity
+   Client.EVM2AnyMessage memory message = Client.EVM2AnyMessage({
+       receiver: abi.encode(bob),
+       data: abi.encode(""),
+       tokenAmounts: tokensToSendDetails,
+       extraArgs: Client._argsToBytes(Client.EVMExtraArgsV1({gasLimit: 0})),
+       feeToken: address(linkToken)
+   });
+   ```
+
+1. Calculate the required fees for the transfer and approve the router to spend LINK tokens for these fees:
+
+   ```solidity
+   uint256 fees = router.getFee(destinationChainSelector, message);
+   linkToken.approve(address(router), fees);
+   ```
+
+1. Send the CCIP transfer request to the router:
+
+   ```solidity
+   router.ccipSend(destinationChainSelector, message);
+   ```
+
+1. Stop impersonating Alice:
+
+   ```solidity
+   vm.stopPrank();
+   ```
+
+1. Record the final token balances of Alice and Bob:
+
+   ```solidity
+   uint256 balanceOfAliceAfter = ccipBnMToken.balanceOf(alice);
+   uint256 balanceOfBobAfter = ccipBnMToken.balanceOf(bob);
+   ```
+
+1. Verify that Alice's balance has decreased by the amount sent and Bob's balance has increased by the same amount:
+
+   ```solidity
+   assertEq(balanceOfAliceAfter, balanceOfAliceBefore - amountToSend);
+   assertEq(balanceOfBobAfter, balanceOfBobBefore + amountToSend);
+   ```
+
+### Test case 2: Transfer with native gas fees
+
+The `test_transferTokensFromEoaToEoaPayFeesInNative` function tests the transfer of tokens between two externally owned accounts (EOA) while paying fees in native gas. Here are the steps involved in this test case:
+
+1. Invoke the `prepareScenario()` function to set up the necessary variables. (This step is the same as in the previous test case.)
+1. Record the initial token balances of Alice and Bob. (This step is the same as in the previous test case.)
+1. Begin impersonating Alice and provide her with native gas to pay for the fees:
+
+   ```solidity
+   vm.startPrank(alice);
+   deal(alice, 5 ether);
+   ```
+
+1. Construct the [Client.EVM2AnyMessage](/ccip/api-reference/client#evm2anymessage) structure. This step is the same as in the previous test case. The main difference is that the `feeToken` is set with `address(0)` to indicate that the fees are paid in native gas:
+
+   ```solidity
+   Client.EVM2AnyMessage memory message = Client.EVM2AnyMessage({
+       receiver: abi.encode(bob),
+       data: abi.encode(""),
+       tokenAmounts: tokensToSendDetails,
+       extraArgs: Client._argsToBytes(Client.EVMExtraArgsV1({gasLimit: 0})),
+       feeToken: address(0)
+   });
+   ```
+
+1. Calculate the required fees for the transfer and send the CCIP transfer request along with the necessary native gas:
+
+   ```solidity
+   uint256 fees = router.getFee(destinationChainSelector, message);
+   router.ccipSend{value: fees}(destinationChainSelector, message);
+   ```
+
+1. Stop impersonating Alice and verify Alice (sender) and Bob (receiver) token balances. (This step is the same as in the previous test case.)
```

Comment:
On a second read, it looks like we're highlighting the few differences between native and LINK; since it's just an "examine the code" explanation section and not a set of steps for users to complete, it's not a blocker to leave this as-is